pkg/asset/installconfig/ovirt/validation.go

```diff
@@ -24,9 +24,11 @@ func Validate(ic *types.InstallConfig) error {
 			"validation requires a Engine platform configuration").Error())
 	}
 
+	testConfig := types.InstallConfig{}
+	testConfig.ObjectMeta.Name = "test"
 	allErrs = append(
 		allErrs,
-		validation.ValidatePlatform(ic.Platform.Ovirt, ovirtPlatformPath)...)
+		validation.ValidatePlatform(ic.Platform.Ovirt, ovirtPlatformPath, &testConfig)...)
```

Comment:
Don't you want to validate the **actual** install config here?
```suggestion
		validation.ValidatePlatform(ic.Platform.Ovirt, ovirtPlatformPath, ic)...)
```

Outside of the scope of this PR, this validation does not even need to be done here. This duplicates validation that has already been done [here](https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/installconfig.go#L149).

Reply:
oh okay.

Reply:
This has been marked as resolved, but the code is still using a test InstallConfig instead of the real InstallConfig.